### DISCUSSION TITLE
Welcome to Youtube-View-Bot Discussions!

### DISCUSSION THREAD

**Discussion by JijaProGamer** [JijaProGamer/youtubeWatchBot#3] [Announcements]
## 👋 Welcome!
  * Ask questions about the bot
  * Share ideas.
  * Engage with other community members.

**Comment by cwhawley** [comment]
how can i learn more about your proxy service? EDIT: i see that it's not launched, so maybe i'll just wait.

**Comment by Potrachen0** [comment]
это программа не работает (  есть детальная инструкция ?
